### PR TITLE
[MI-3475] Add feature to unlink linked channels

### PR DESCRIPTION
**Pull request by ayusht2810**
#### Summary
Add a feature to unlink linked channels
[screen-capture (31).webm](https://github.com/mattermost/mattermost-plugin-msteams-sync/assets/100013900/759d8fcf-9f5f-45a4-bd9e-84b08cb39103)

### CODE REVIEW

File: webapp/src/containers/Rhs/index.tsx [hunk 1]
```diff
@@ -258,13 +302,20 @@ const Rhs = (): JSX.Element => {
             <Dialog
                 description={dialogContent}
                 destructive={showDestructivetDialog}
-                show={showDisconnectDialog}
-                primaryActionText={showDestructivetDialog && 'Disconnect'}
-                onCloseHandler={() => setShowDisconnectDialog(false)}
-                onSubmitHandler={showDestructivetDialog && disconnectUser}
+                show={showDialog}
+                primaryActionText={showDestructivetDialog && primaryActionText}
```
Comment (raghavaggarwal2308) [inline webapp/src/containers/Rhs/index.tsx:306]:
```suggestion
                primaryActionText={showDestructiveDialog && primaryActionText}
```

File: webapp/src/containers/Rhs/index.tsx [hunk 2]
```diff
@@ -104,8 +144,9 @@ const Rhs = (): JSX.Element => {
                             className='rhs-disconnect__disconnect-button'
                             onClick={() => {
                                 setDialogContent('Are you sure you want to disconnect your MS Teams Account?');
-                                setShowDisconnectDialog(true);
+                                setShowDialog(true);
                                 setShowDestructiveDialog(true);
+                                setPrimaryActionText('Disconnect');
```
Comment (raghavaggarwal2308) [inline webapp/src/containers/Rhs/index.tsx:149]: Constants for 'Unlink' and 'Disconnect' as they are being used at multiple places

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]
Fix CI

**Comment by ayusht2810** [comment]
Closing this PR in favour of updated [figma designs](https://www.figma.com/file/aRgXgPZQ3lDaow5uatDzz9/MS-Teams-Sync-RHS?type=design&node-id=1-7&mode=design&t=ewtCZk3sFs0sMBLl-0). We will pick changes from this PR if required.

**Pull request closed without merging**